src/components/Layout/DrawerNavigation.tsx

```diff
@@ -118,18 +118,30 @@ export const DrawerNavigationComponent: FunctionComponent<
   const onPressCloseDrawer = (): void => {
     navigation.dispatch(DrawerActions.closeDrawer());
   };
-  const releaseChannel: string | undefined = Constants.manifest?.releaseChannel;
```

Comment:
as mentioned in https://github.com/rationally-app/mobile-application/pull/660#discussion_r991333193, release channel isnt found in manifest anymore